### ISSUE TITLE
wasm instantiation fails for pyodide engine

### ISSUE THREAD

**Issue by statik** [r-wasm/quarto-drop#6]
I love this extension!

I'm trying to use drop on a revealjs presentation that is in a subdirectory of a quarto website project. It works fine with the webr engine, but the pyodide engine fails to instantiate due to a javascript exception. I wasn't able to track down exactly what is going wrong, but I was able to put together a minimal reprex at https://github.com/statik/reprex-drop.

If you clone that repo and run quarto preview, you should be able to click into the presentation, try opening the drop console, and look at the browser developer tools to see the exception. I am running this on Chrome Version 126.0.6478.183 (Official Build) (arm64) on macOS. Thanks for your help!

**Comment by statik** [comment]
<img width="1717" alt="Screenshot 2024-07-24 at 10 18 30 AM" src="https://github.com/user-attachments/assets/8ae9da2b-6d7a-41e4-b6b3-4e365f1cfc54">

**Comment by statik** [comment]
huh, I'm actually seeing pyodide fail when running `quarto preview python-example.qmd` from this repo directly.

**Comment by georgestagg** [comment]
Hmmm, strange!

The forbidden response is expected for that path, but can you access the full URL at https://cdn.jsdelivr.net/pyodide/v0.26.1/full/pyodide.asm.js in Chome? What about Safari?

I can't reproduce at my end. This is looking like it might be some kind of network issue, but I'm unsure.

Could you please look for `pyodide.asm.js` (or some other file under the above URL) in your Chrome dev tools network log and show me the details of that particular connection? Here is what a good connection should look like:

![Screenshot 2024-07-25 at 09 51 44](https://github.com/user-attachments/assets/744016fa-2718-4291-aa9e-fc23299cdc8c)

This problem might also be avoided with #4, but that will require some work to implement.

**Comment by statik** [comment]
thank you for looking at this! I started working on creating a HAR recording to reproduce, and did that in an incognito window. Things started working! I then went through and disabled some Chrome Extensions, and pyodide works now. I did see that even when the initialization was failing, pyodide.asm.js did successfully fetch over the network.

It turns out that [Mokku Chrome extension](https://chromewebstore.google.com/detail/mokku/llflfcikklhgamfmnjkgpdadpmdplmji) was breaking things. I have removed that extension.
